simtools/ray_tracing.py

```diff
@@ -249,103 +251,107 @@ def analyze(
-                    image.centroidX = centroidX
-                    image.centroidY = centroidY
-                    image.setEffectiveArea(effArea * telTransmission)
+                    containment_diameter_deg = containment_diameter_cm * cm_to_deg
+                    image.set_psf(containment_diameter_cm, fraction=containment_fraction, unit="cm")
+                    image.centroid_x = centroid_x
+                    image.centroid_y = centroid_y
+                    image.set_effective_area(eff_area * tel_transmission)
                 else:
-                    containment_diameter_cm = image.getPSF(containment_fraction, "cm")
-                    containment_diameter_deg = image.getPSF(containment_fraction, "deg")
-                    centroidX = image.centroidX
-                    centroidY = image.centroidY
-                    effArea = image.getEffectiveArea() * telTransmission
-
-                effFlen = np.nan if thisOffAxis == 0 else centroidX / tan(thisOffAxis * pi / 180.0)
-                _currentResults = (
-                    thisOffAxis * u.deg,
+                    containment_diameter_cm = image.get_psf(containment_fraction, "cm")
+                    containment_diameter_deg = image.get_psf(containment_fraction, "deg")
+                    centroid_x = image.centroid_x
+                    centroid_y = image.centroid_y
+                    eff_area = image.get_effective_area() * tel_transmission
+
+                eff_flen = (
+                    np.nan if this_off_axis == 0 else centroid_x / tan(this_off_axis * pi / 180.0)
+                )
+                _current_results = (
+                    this_off_axis * u.deg,
                     containment_diameter_cm * u.cm,
                     containment_diameter_deg * u.deg,
-                    effArea * u.m * u.m,
-                    effFlen * u.cm,
+                    eff_area * u.m * u.m,
+                    eff_flen * u.cm,
                 )
-                if self.config.singleMirrorMode:
-                    _currentResults += (thisMirror,)
-                _rows.append(_currentResults)
+                if self.config.single_mirror_mode:
+                    _current_results += (this_mirror,)
+                _rows.append(_current_results)
 
-        if doAnalyze:
+        if do_analyze:
             _columns = ["Off-axis angle"]
             _columns.extend(list(self.YLABEL.keys()))
-            if self.config.singleMirrorMode:
+            if self.config.single_mirror_mode:
                 _columns.append("mirror_number")
             self._results = QTable(rows=_rows, names=_columns)
 
-        self._hasResults = True
+        self._has_results = True
         if export:
-            self.exportResults()
+            self.export_results()
 
-    def _processRX(self, file, containment_fraction=0.8):
+    def _process_rx(self, file, containment_fraction=0.8):
```

Comment:
'rx' is kept lowercase here but I've seen before in the code it in uppercase after the underscore. I guess we have to decide which one to use and keep the same throughout the code.

Reply:
You are right, I contemplated while doing it as well. I think I will change everywhere to be rx.